lib/models/utils_models.dart

```diff
@@ -0,0 +1,10 @@
+class EnumValues<T> {
```

Comment:
I believe I know what you want to do with that (Strings in an Enum) but rather that I would recommend to write an extension method: https://stackoverflow.com/a/58552304/3690032

Reply:
Thanks, @md-weber I will review the documentation :)